app/src/main/res/values/strings.xml

```diff
@@ -60,4 +60,7 @@
     <string name="_speaker">*speaker*</string>
     <string name="_date">*date*</string>
 
+    <!-- Intent item key strings -->
+    <string name="intent_word_id_key">word id</string>
```

Comment:
What is this for?

Reply:
Left over from a different attempt to get the word id
Will remove